renormalizer/model/phonon.py

```diff
@@ -161,7 +154,7 @@ def e0(self):
 
     @property
     def is_simple(self):
-        return self.force3rd == (0.0, 0.0) and self.nqboson == 1 and self.omega[0] == self.omega[1]
+        return self.force3rd == (0.0, 0.0)  and self.omega[0] == self.omega[1]
```

Comment:
I think the force3d is not needed anymore.

Reply:
It is now only used in TDH module. This feature will be removed along with the module.